### DISCUSSION TITLE
Audacious takes 1 minute 24 seconds to load on windows

### DISCUSSION THREAD

**Discussion by NolonQ** [audacious-media-player/audacious#86] [Q&A]
This is frustrating, audacious looks like the near perfect audioplayer and i enjoy it alot but this waiting times to open the program are a pain.
I recorded my screen, opened audacious and waited, after 1minute 24 seconds the software GUI appeared on my screen.
I tried creating an account on the audacious forums but i recieve no confirmation email even after requesting one for the third time... so that's how i ended up here seeking for help.

Note that i heavily modified this windows 10 21H2 system and that could potentialy be the reason behind the slow loading times, however i have no problems with any other programs.
Are there any windows-services audacious depends on?

**Comment by jlindgren90** [comment]
It doesn't take anywhere near that long on any Windows system I've tried. Maybe a few seconds on first start.

**Reply by Lascaille** [reply]
Sorry to cross-thread here but I asked on another thread what the correct way is to format replaygain info for aac files, as it's not picked up by the m4a reader. Did you happen to see it?

**Comment by NolonQ** [comment]
Alright thanks for the info.
Any suggestions what i should try to fix it? What about dependencie services?
When i klick on the audacious desktop shortcut i can see the process immediately starting in taskmanager together with gdbus.exe
but as i said it takes over a minute for the GUI to show on screen.

I might have found the cause tho, i use portmaster from safing.io and by default all connections are blocked.
I see that portmaster blocked connection requests from audacious, those are:
127.0.0.1 :50074
::1 :50074
These seem like local connections to me, but i have no idea what the port 50074 is.

I've got another question, my keyboard has mediakeys for pause, resume, skip, etc and these used to work with windowsmediaplayer but audacious does not react to any of these keys, how do i configure them? Currently the only way for me to pause/resume/skip is to tab into audacious and use the GUI.

Edit: As i thought blocking these local connections caused the slow loading times.
Turns out gdbus.exe also uses some local connections.
127.0.0.1 :50151
127.0.0.1 :50149
127.0.0.1 :50137
127.0.0.1 :50135
::1 :50150
::1 :50148
::1 :50136
::1 :50154

I've noticed that when closing audacious, the gdbus.exe service remains active in taskmanager, why is that why doesn't it terminate itself when audacious is closed?

**Reply by jlindgren90** [reply]
GDBus provides the D-Bus session bus (an IPC mechanism) and typically remains until logout, I believe. If you want to know more, https://en.m.wikipedia.org/wiki/D-Bus is a good place to start.
